src/BuildScriptGenerator/Constants.cs

```diff
@@ -20,7 +20,7 @@ public static class Constants
         public const string True = "true";
         public const string False = "false";
 
-        public const string TemporaryInstallationDirectoryRoot = "/tmp/oryx/platforms";
+        public const string TemporaryInstallationDirectoryRoot = "/opt/";
```

Comment:
that does not look right..  why do you want to change this

Reply:
I thought about this more and could not find a real benefit to us or users installing platforms in `/tmp/oryx/platforms`. `/opt` makes more sense since that is where add-on application software is usually stored. 

Do you foresee this being a breaking change?

Reply:
I noticed this caused various pipeline issues like `permissions denied` so I reverted back to original location